### DISCUSSION TITLE
[4.0] List status filtering options applied to new content (articles, modules)

### DISCUSSION THREAD

**Discussion by crommie** [joomla/joomla-cms#34970] [Q&A] [answered]
I don't know if this is a bug or expected behaviour, but: if you use Filter Options for Status in your article or module list and then create a new item, the status is set to whatever you chose as your filter option. 

### Steps to reproduce the issue

In the backend menu, click on Content -> Articles.

Set Filter Options to Status -> Trashed (or Archived, or Unpublished).

Create a new article.

### Expected result

I would expect the Status to be Published (since that's the default when you create content).

### Actual result

Status is set to Trashed (or whatever you chose in your filter options).

This also applies to Modules.

### System information (as much as possible)

Joomla 4.0.0-rc5

### Additional comments

Maybe this has been reported already (I couldn't find it). And maybe, like I said, it's expected - if so: it is a bit confusing but maybe that's just me.

**Comment by brianteeman** [comment, marked as answer]
It's always been this way. Check j3 :)

**Comment by crommie** [comment]
Seriously?

**Comment by crommie** [comment]
Seriously. I just checked. Thanks @brianteeman 

It's still confusing, but somehow it's kind of reassuring to know that I've never noticed this before.

**Comment by rdeutz** [comment]
makes sense not in all cases, but for categories and when you want to write some articles that are unpublished it makes.

**Reply by brianteeman** [reply]
I knew because I use it quite a bit
